src/WCSimPhysicsList.cc

```diff
@@ -522,66 +551,55 @@ void WCSimPhysicsList::ConstructHad()
 	  // Choice defined in jobOptions.mac, which is
 	  // read in before initialization of the run manager.
 	  // In the absence of this file, BINARY will be used.
-	  // Gheisha = Original Geant4 default 
+	  // Gheisha = Original Geant4 default // DISAPPEARED IN G4.10
 	  // Bertini = Bertini intra-nuclear cascade model
 	  // Binary  = Binary intra-nuclear cascade model
-	  if (gheishahad) {
-	    G4LEProtonInelastic* theLEInelasticModel = new G4LEProtonInelastic;
-	    theInelasticProcess->RegisterMe(theLEInelasticModel);
-	  }
-	  else if (bertinihad) {
-	    G4CascadeInterface* theBertiniModel = new G4CascadeInterface;
-	    theInelasticProcess->RegisterMe(theBertiniModel);
+	  if (bertinihad) {
+	   G4CascadeInterface* theBertiniModel = new G4CascadeInterface;
+	   theBertiniModel->SetMaxEnergy(5.0*GeV);
```

Comment:
This values seems to be the one by default for the FTFP_BERT Physics List.
See http://geant4.cern.ch/support/proc_mod_catalog/physics_lists/hadronic/FTFP_BERT.html